scripts/container.sh

```diff
@@ -15,7 +15,7 @@ if [[ $* == *--build* ]]; then
         --rm
 fi
 
-if [[ $* == *--clear_cache* ]]; then
+if [[ $* == *--clear_cache* ]] || [[ $* == *--clear-cache* ]]; then
```

Comment:
What does this do? Why the duplication?

Reply:
Nevermind I see it. Lets remove the `clear_cache argument` and stick with one command rather than two similar arguments